Sources/GRPC/ServerCallContexts/StreamingResponseCallContext.swift

```diff
@@ -80,7 +80,7 @@ open class StreamingResponseCallContextImpl<ResponseMessage: Message>: Streaming
 /// Concrete implementation of `StreamingResponseCallContext` used for testing.
 ///
 /// Simply records all sent messages.
-open class StreamingResponseCallContextTestStub<ResponseMessage: Message>: StreamingResponseCallContext<ResponseMessage> {
+public class StreamingResponseCallContextTestStub<ResponseMessage: Message>: StreamingResponseCallContext<ResponseMessage> {
```

Comment:
If you make this public, the members should be, as well. However, I had made this `open` to let library users override individual methods if needed for more complex tests. WDYT?

Reply:
If we make it `open` then any super classes must be `open` as well. Maybe that's not actually a problem here since the API is very small (I'd be okay dropping this PR).

Reply:
Hmm, not sure either way. We could make it `public` for now and add a comment that we would be open (sic!) to making it `open` if requested? I.e. they should file an issue about this if they need it?

Reply:
Let's hold off on this change for now, given https://github.com/grpc/grpc-swift/pull/657#discussion_r358270353 this might become irrelevant.